Python/Product/PythonTools/visualstudio_py_debugger.py

```diff
@@ -2468,7 +2465,7 @@ def _excepthook(exc_type, exc_value, exc_tb):
             write_bytes(conn, LAST)
         last_ack_event.wait(5)
 
-    if wait_on_exit:
+    if wait_on_normal_exit in debug_options:

```

Comment:
Can't see all the scopes in between, but `wait_on_normal_exit` here will be a bool or a name error, and not in the set.